src/documentation/pages/components/footer/examples/_external-lumen-portal.vue

```diff
@@ -0,0 +1,530 @@
+                li
+                  a.optanon-toggle-display(href='#') Cookie Settings
+                <!-- OneTrust Cookies Settings button end -->
+                li
+                  a(
+                    href='https://www.centurylink.com/aboutus/legal.html',
+                    target='_blank'
+                  ) Legal
+                li
+                  a(
+                    href='https://www.lumen.com/en-us/about/legal/acceptable-use-policy.html',
+                    target='_blank'
+                  ) Legal Notices
+                li
+                  a(
+                    href='https://www.lumen.com/en-us/about/legal/privacy-notice.html'
+                  ) Privacy
+                li
+                  a(
+                    href='https://www.centurylink.com/aboutus/company-information/public-policy.html',
+                    target='_blank'
+                  ) Public Policy
+                li
+                  a(
+                    href='https://www.lumen.com/help/en-us/home.html',
+                    target='_blank'
+                  ) Support
+                li
+                  a(
+                    href='https://www.lumen.com/en-us/about/legal/commission-for-complaints-for-telecom-television-services-ccts'
+                  ) Canadian Ombudsman
+                li
+                  a(
+                    href='https://assets.lumen.com/is/content/Lumen/gender-pay-gap-report?Creativeid=b565bf28-6d55-4f86-a245-2f0eb40b91a3',
+                    target='_blank'
+                  ) Gender Pay Gap 2020 (UK)
+                li
+                  a(
+                    href='https://www.lumen.com/es-ar/about/legal/latam-tariffs-regulatory.html'
+                  ) Legal (Latin America)
+                li
+                  a(
+                    href='https://www.lumen.com/en-us/about/legal/de-legal-notices.html'
+                  ) Legal Notices (Germany)
+                li
+                  a(
+                    href='https://assets.lumen.com/is/content/Lumen/modern-slavery-statement-1?Creativeid=5dbb9687-e4a8-4a40-ae00-673ce51d6a80',
+                    target='_blank'
+                  ) Modern Slavery Statement 2020 (UK)
+                li
+                  a(
+                    href='https://www.lumen.com/en-us/about/legal/uk-tax-strategy.html'
+                  ) UK Tax Strategy
+              .chi-footer__copyright
+                | &copy; 2022 Lumen Technologies. All Rights Reserved.
+                | Lumen is a registered trademark in the United States, EU and certain other countries.
+  </Wrapper>
+  <pre class="language-html" slot="code-webcomponent">
+    <code v-highlight="$data.codeSnippets.webcomponent" class="html"></code>
+  </pre>
+  <Wrapper slot="code-htmlblueprint">
+    <JSNeeded />
+    <pre class="language-html">
+      <code v-highlight="$data.codeSnippets.htmlblueprint" class="html"></code>
+    </pre>
+  </Wrapper>
+</ComponentExample>
+</template>
```

Comment:
Check this identation in every example.

```suggestion
  </ComponentExample>
</template>
```